scripts/prepare_ci_kafka.sh

```diff
@@ -35,7 +37,7 @@ for filename in "$SCRIPT_PATH"/test_data/*; do
     if [[ "$topic" = *bin ]]; then
         kafkacat -P -b 127.0.0.1:29092 -t "$topic" "$filename"
     else
-        cat "$filename" | kafkacat -P -b 127.0.0.1:29092 -t "$topic"
+        kafkacat -P -b 127.0.0.1:29092 -t "$topic" < "$filename"
```

Comment:
why propose this change?

Reply:
It's because I encountered this shellcheck warning in one of the builds:
https://buildkite.com/risingwavelabs/pull-request/builds/10512#0183ef4f-04ed-4cd8-a477-928d4479d76f